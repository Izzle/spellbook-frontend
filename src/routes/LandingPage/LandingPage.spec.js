import React from 'react';
import renderer from 'react-test-renderer';
import ReactDOM from 'react-dom';
import LandingPage from './LandingPage';
import { BrowserRouter } from 'react-router-dom';


describe('<LandingPage />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<BrowserRouter><LandingPage /></BrowserRouter>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders this UI as expected', () => {
        const tree = renderer.create(<BrowserRouter><LandingPage /></BrowserRouter>).toJSON();
        expect(tree).toMatchSnapshot();
    });    
});