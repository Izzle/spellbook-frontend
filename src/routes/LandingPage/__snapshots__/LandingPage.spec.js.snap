// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage /> renders this UI as expected 1`] = `
<section
  className="LandingPage"
>
  <h2>
    What is SpellBook?
  </h2>
  <p>
    It's a tool to keep track of your spells in Dungeons and Dragons. You can view and read up on spells, create your own custom spells, and add spells to different spellbooks for each character you play.
  </p>
  <h2>
    How do I make a spellbook?
  </h2>
  <p>
    First select 'View Spellbooks' then select the book you would like to view. From there, you can click 'Edit Spellbook' to see all available spells. Any spells currently in your Spellbook will be highlighted, just click on any spell to add or remove it.
  </p>
  <p>
    When you are done making your selections, click 'Save Spellbook' to save your spellbook for future use.
  </p>
  <h2>
    How do I make a new spell?
  </h2>
  <p>
    Click on 'Create a Spell' in the navigation, fill out all information about the spell such as its name, range, and duration. When you're done, hit 'Create Spell' and your spell will be added to all the spells in our library. From then on, you can add your custom spell to any spellbook!
  </p>
</section>
`;
